pkg/operator/starter.go

```diff
@@ -0,0 +1,102 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+
+	dynamicclient "k8s.io/client-go/dynamic"
+	kubeclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/klog"
+
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/library-go/pkg/operator/csi/csicontrollerset"
+	goc "github.com/openshift/library-go/pkg/operator/genericoperatorclient"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+
+	"github.com/ovirt/csi-driver-operator/pkg/apis/operator/v1alpha1"
+	"github.com/ovirt/csi-driver-operator/pkg/generated"
+)
+
+const (
+	// Operand and operator run in the same namespace
+	defaultNamespace = "openshift-cluster-csi-drivers"
+	operatorName     = "ovirt-csi-driver-operator"
+	operandName      = "ovirt-csi-driver"
+)
+
+func RunOperator(ctx context.Context, controllerConfig *controllercmd.ControllerContext) error {
+	// Create clientsets and informers
+	kubeClient := kubeclient.NewForConfigOrDie(rest.AddUserAgent(controllerConfig.KubeConfig, operatorName))
+	dynamicClient := dynamicclient.NewForConfigOrDie(rest.AddUserAgent(controllerConfig.KubeConfig, operatorName))
+	kubeInformersForNamespaces := v1helpers.NewKubeInformersForNamespaces(kubeClient, defaultNamespace, "")
+
+	// Create GenericOperatorclient. This is used by the library-go controllers created down below
+	gvr := v1alpha1.SchemeGroupVersion.WithResource("ovirtcsidrivers")
+	operatorClient, dynamicInformers, err := goc.NewClusterScopedOperatorClient(controllerConfig.KubeConfig, gvr)
```

Comment:
Please use `NewClusterScopedOperatorClientWithConfigName` instead; it has has been merged into library-go recently.

Reply:
ack